ccswbs/gus
# Release/1.0.0: Update page title responsiveness and maintenance fixes

# Summary of changes
- Fixes an issue with the hero image caption/page title getting cutoff on smaller displays.
- Apply dependabot maintenance fixes

## Frontend
- Style changes in hero.css
- Apply dependabot maintenance fixes

[x] My changes are accessible (at minimum WCAG 2.0 Level AA)
[x] My changes are responsive and appear as expected on mobile and desktop views.
[n/a] After merging, I will update the [Content Hub documentation](https://uoguelphca.sharepoint.com/sites/UniversityContentHubInformationGroup) and ensure the Content Hub trainer understands how my changes will affect users.

## Backend
None

# Test Plan

1. Review https://deploy-preview-202--ugconthub.netlify.app/ for issues
2. Go to any page with a banner image (ex. https://deploy-preview-202--ugconthub.netlify.app/international/events/) on a smaller display, ensure the page title doesn't get cutoff
3. Ensure menus are showing on https://deploy-preview-202--ugconthub.netlify.app/studentexperience
